cardano-db-sync/src/Cardano/DbSync/Era/Allegra/Util.hs

```diff
@@ -215,8 +215,8 @@ txInvalidHereAfter :: Shelley.Tx StandardAllegra -> Maybe Word64
 txInvalidHereAfter =
```

Comment:
```suggestion
txInvalidHereafter =
```

Reply:
I detected the casing error while mapping the column in `cardano-graphql`

At this stage I'll be remapping as
```
tx.invalidHereAfter AS "invalidHereafter"
```
... , however it would be best to fix now for consistency.

Reply:
PR is https://github.com/input-output-hk/cardano-db-sync/pull/417 .